src/main/resources/public/jsx/DatasetForm.jsx

```diff
@@ -18,6 +18,8 @@ class DatasetForm extends React.Component {
         notes: React.PropTypes.string.isRequired,
         licenses: React.PropTypes.object.isRequired,
         license: React.PropTypes.string.isRequired,
+        organization:React.PropTypes.string.isRequired,
+        organizations:React.PropTypes.object.isRequired,
```

Comment:
An array is provided
![image](https://user-images.githubusercontent.com/8415401/27333248-861cb5a8-55c5-11e7-988d-5133405194d9.png)